openlane/steps/verilator.py

```diff
@@ -158,6 +163,10 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
         if self.config["LINTER_ERROR_ON_LATCH"]:
             extra_args.append("--Werror-LATCH")
 
+        if include_dirs := self.config["VERILOG_INCLUDE_DIRS"]:
+            include_flags = ["-I" + str(dir) for dir in include_dirs]
```

Comment:
The way you've coded this, all the includes would be one argument, which, so if you have two directories, it would be:

```
-Idir1 -Idir2
```

This would include the directory literally named `dir1 -Idir2`, which is neither directory.

This can be fixed pretty easily: Replace 167, 168 with just this.

```suggestion
            extra_args.extend([f"-I{dir}" for dir in include_dirs])
```

Reply:
Nice change, implemented!